mllib/src/main/scala/org/apache/spark/ml/feature/Word2Vec.scala

```diff
@@ -274,6 +274,29 @@ class Word2VecModel private[ml] (
     wordVectors.findSynonyms(word, num)
   }
 
+  /**
```

Comment:
actually I tried call the `findSynonymsArray` from python, but for `findSynonymsArray()` I got below in python, which has no data, 
```
>>> model.findSynonymsArray("a", 2)
[{u'__class__': u'scala.Tuple2'}, {u'__class__': u'scala.Tuple2'}]
```
which I posted a bit long ago https://github.com/apache/spark/pull/17451#issuecomment-290951029
that's why I switched to create a new method in scala